generated_cpp_50/com_trolltech_qt_core/com_trolltech_qt_core0.cpp

```diff
@@ -50,7 +50,7 @@ PythonQtShell_QAbstractAnimation::~PythonQtShell_QAbstractAnimation() {
 void PythonQtShell_QAbstractAnimation::childEvent(QChildEvent*  arg__1)
```

Comment:
I wonder if we should remove these older versions of generated code now, instead of adapting them to these changes each time. 
Qt versions before 5.9 are not tested at all, and we do not test the checked-in generated code anyway, so we cannot guarantee that it will still work with the current version of PythonQt. Had also a discussion with @usiems about this.